vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -926,6 +929,16 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
             && validSelectionModes.indexOf(mode) >= 0) {
           selectionMode = mode;
           selectedKeys = {};
+          if (mode === validSelectionModes[0]) { // SINGLE
+            grid.$.table.setAttribute('aria-multiselectable', false);
+          } else if (mode === validSelectionModes[1]) { // NONE
+            grid.$.table.removeAttribute('aria-multiselectable');
+            // TODO: remove aria-selected from all rows
+          } else { // MULTI
+            grid.$.table.setAttribute('aria-multiselectable', true);
+          }
+          // re-run renderers for all items visible
+          grid.requestContentUpdate();
```

Comment:
All the new tests pass even with this line removed. Does something else cause an item update on dynamic selection mode change?